### PR TITLE
Develop.sprint6.separate calls

### PR DESCRIPTION
**Pull request by IegorKozakov**
*Separate audio and video call buttons

### CODE REVIEW

File: samples/webrtc/js/app.js [hunk 1]
```diff
@@ -382,7 +401,7 @@
 
                                 app.helpers.stateBoard.update({'title': 'calling'});
 
-                                document.getElementById(sounds.call).play();
+                                // document.getElementById(sounds.call).play();
```
Comment (dimaspirit) [inline samples/webrtc/js/app.js]: Why do you need commented?

File: samples/webrtc/js/app.js [hunk 2]
```diff
@@ -416,7 +435,19 @@
 
         /** ACCEPT */
         $(document).on('click', '.j-accept', function() {
+            isAudio = app.currentSession.callType === 2;
```
Comment (dimaspirit) [inline samples/webrtc/js/app.js]: Better use const 
QB.webrtc.CallType.VIDEO

File: samples/webrtc/js/app.js [hunk 3]
```diff
@@ -358,10 +358,23 @@
                 }
 
                 app.helpers.stateBoard.update({'title': 'create_session'});
-                app.currentSession = QB.webrtc.createNewSession(Object.keys(app.callees), QB.webrtc.CallType.VIDEO);
 
+                isAudio = this.getAttribute('data-call') === 'audio';
```
Comment (dimaspirit) [inline samples/webrtc/js/app.js]: Maybe better idea uses event object? Up to you

File: samples/webrtc/js/app.js (hunk 2 above)
Comment (IegorKozakov) [reply samples/webrtc/js/app.js]: You mean QB.webrtc.CallType.AUDIO. Yes, it's a good idea.

**Pull request merged**